packages/queued-request-controller/CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.13.0]
+
+### Changed
+
+- **BREAKING:** Bump peer dependency `@metamask/selected-network-controller` to `^14.0.0` ([#4374](https://github.com/MetaMask/core/pull/4374))
```

Comment:
This was also done in 0.12.0 — in fact the peer dependency is up to `^15.0.0` now.

It looks like we are bumping the `devDependency` in this PR to match the new release of `selected-network-controller`. I am not sure whether it should be a dev dependency, but that is what it is today. So perhaps we do not need a new release of `queued-request-controller`? Or if that seems wrong, perhaps we ought to move it from `devDependencies` to `dependencies` in another PR and then we can make a new release of `queued-request-controller` that makes use of the new version of `selected-network-controller`.

Reply:
oh sure enough only types are imported here... So yeah maybe we don't need to release?

Reply:
Yeah, you may not need to. You might try upgrading the extension to use the latest version of these packages and seeing what happens.

Reply:
OK yeah I don't think we need to release `QueuedRequestController`